scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -83,11 +72,13 @@ class ScanCode(
     name: String,
     config: ScannerConfiguration
 ) : LocalScanner(name, config) {
```

Comment:
The tests could also be moved to their own file

Reply:
Moved the tests affected to a new test class ScanCodeResultParserTest.